tests/test-awful-mouse.lua

```diff
@@ -0,0 +1,41 @@
+require("awful.mouse")
+
+local steps = {}
+
+-- warning: order is important
+local buttons = {"left", "middle", "right"}
+
+-- Check the is_****_mouse_button_pressed properties.
+for k, v in ipairs(buttons) do
```

Comment:
To be honest, I have so little time these days that I would rather just merge this as-is. Feel free to propose a PR to improve this, but I just don't have the bandwidth to revisit this right now instead of attempting to fix more bugs :(

Reply:
That's fine. It was intended as a nice-to-have anyway, hence the approval.